utils/handler.ts

```diff
@@ -34,12 +34,6 @@ export const handleSearchKeyPress = async (e: React.KeyboardEvent<HTMLInputEleme
     return
   }
 
-  if (Number.isNaN(+search)) {
```

Comment:
Why not check if the `search` is a hash or block number before the request?

Reply:
I've got it

> fix redirect to tokens list logic, if token name is matched, go to token details page directly

But the solution is improper because there could be many tokens having a same `token name` while the API only returns the first one, which is misleading

Reply:
@zmcNotafraid we'd better remove the function of searching by token name, or return a list instead of the first record

Reply:
The API will be updated along with https://github.com/Magickbase/godwoken_explorer/issues/1159 and a new page of matched token list will be added in the future.

Now just let's just recover the checking of `token name` before sending the request @qiweiii